EAD_Session7/com/metacube/ead/configuration/JavaConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.metacube.ead.configuration;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+import com.metacube.ead.service.implementation.MockMailSender;
+import com.metacube.ead.service.implementation.SmtpMailSender;
+
+/**
+ * The Class JavaConfig is basically used to set the configuration
+ * of the user defined Bean in the IOC container 
+ * 
+ * @author Aman Gautam
+ * 
+ * Dated :11/9/19.
+ */
+@Configuration
+@ComponentScan(basePackages = "com.metacube.ead")
+public class JavaConfig {
+
+//  @Qualifier("smtp")
+//	@Primary
+//	@Bean
```

Comment:
Since everything is committed then why it is still here

Reply:
Sir this is because , functionality and annotations keep on changing so , i have commented so that we can choose the annotations as per requirenment .